development/build/index.js

```diff
@@ -132,6 +132,8 @@ async function defineAndRunBuildTasks() {
         'stateHooks',
         'sentryHooks',
         'sentry',
+        'JSON',
```

Comment:
Hmm. This will have a significant impact upon the production bundle. Making this change in a PR that appears to be test-focused is potentially misleading.

When creating a changelog, we might reasonably look at this PR and conclude no functional changes were made.